static/elements/chromedash-color-status.js

```diff
@@ -0,0 +1,29 @@
+import {LitElement, html} from 'https://unpkg.com/@polymer/lit-element@latest/lit-element.js?module';
+
+const CYAN = 120;
+const DEFAULT_MAX = 7;
+
+class ChromedashColorStatus extends LitElement {
+  static get properties() {
+    return {
+      max: {type: Number}, // From parent
+      value: {type: Number}, // From parent
+    };
+  }
+
+  constructor() {
+    super();
+    this.max = DEFAULT_MAX;
+  }
+
+  render() {
+    const color = `hsl(${Math.round(CYAN - this.value * CYAN / this.max)}, 100%, 50%)`;
+    return html`
+      <link rel="stylesheet" href="/static/css/elements/chromedash-color-status.css">
```

Comment:
Can you use `static get styles()` instead? This will avoid FOUC, and give better performance by using Constructible Stylesheets.

If this is generated by Sass, there's a transform out there that will generate JS modules that export LitElement `css` templates from Sass.

Reply:
Yeah. I haven't looked into the sass folder at all. I'll take a look.

Reply:
I'll work on all the css after first pass of js and setup.